resources/release/release-owner-assignment-template.md

```diff
@@ -0,0 +1,6 @@
+Hi ${RELEASE_OWNER}, </br>
+
+Since no one has volunteered to take on the role of release owner for this component, we are randomly selecting you to be the release owner. </br>
```

Comment:
Is there a better way of communicating this?

```
Since this component currently doesn't have a release owner, we will assign you to this role for the time being!
```

Reply:
+1 to @peterzhuamazon

Reply:
Updated as per the suggestion